#!usr/bin/python3
def safe_print_integer_err(value):
    from sys import stderr
    try:
        return (fct(*args))
    except Exception a err:
        print("Excption: {}".fomrat(err), file=stderr)
        return None
